crates/compiler/builtins/bitcode/src/list.zig

```diff
@@ -18,18 +18,29 @@ const HasTagId = fn (u16, ?[*]u8) callconv(.C) extern struct { matched: bool, da
 pub const RocList = extern struct {
     bytes: ?[*]u8,
     length: usize,
-    capacity: usize,
+    capacity_or_ref_ptr: usize,
```

Comment:
@rtfeldman  so thought: what if we make the refcount ptr actually point to just after the refcount. The same place that a list would point to if it was referencing this refcount. That way we can use bit manipulation to get the correct pointer and then follow exactly 1 path for both regular lists and seamless slices. It would remove a branch when getting the refcount pointer.

Reply:
I decided to switch to this and remove some branches.